vehicle/src/main/scala/com/wedriveu/vehicle/control/VehicleBehavioursImpl.scala

```diff
@@ -27,55 +24,62 @@ trait VehicleBehaviours {
 
 }
 
-class VehicleBehavioursImpl(vehicle: SelfDrivingVehicle) extends VehicleBehaviours {
-   var notRecharging: Boolean = true
+class VehicleBehavioursImpl(selfDrivingVehicle: SelfDrivingVehicle) extends VehicleBehaviours {
    val zeroBattery: Double = 0.0
    val batteryThreshold: Double = 20.0
    val stateRecharging: String = "recharging"
-   val batteryToConsume: Double = 1.0
+   val batteryToConsume: Double = 0.14
```

Comment:
Can you jus write a brief comment explaining why this `val batteryToConsume` has this strange value?